src/main/kubernetes/artemis/deployments/artemis-service-deployment.yml

```diff
@@ -0,0 +1,88 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    field.cattle.io/creatorId: user-gpcv5
+  labels:
+    cattle.io/creator: norman
```

Comment:
Remove Rancher specific Labels! Maybe add an artemis label to allow easy selector usage